build.gradle

```diff
@@ -238,7 +238,7 @@ allprojects {
    */
   test {
     jvmArgs = [
-      '-Xmx2g',
+      '-Xmx4g',
```

Comment:
I think this will increase the memory allocation for all builds (including local).  We may want to restrict this change to just acctests (not entirely sure that's possible).  The concern is if this causes people to run out of system memory when running tests locally...

Reply:
This used to be 4G; and  changed to 2G when I put the PR in for circle about a week or so ago. I've tried testing with 2G but the acceptance ones seem to sporadically exit when the tests go past 2, so am reverting this back to 4G